src/common/Requirements.ts

```diff
@@ -39,11 +39,15 @@ export interface RequirementList {
     executeSetup(): Promise<SetupTestResult>;
 }
 
-// This function wraps existing promises with the intention to allow the collection of promises
-// to settle when used in conjunction with Promise.all(). Promise.all() by default executes until the first
-// rejection. We are looking for the equivalent of Promise.allSettled() which is scheduled for ES2020.
-// Once the functionality is  available  in the near future this function can be removed.
-// See https://github.com/tc39/proposal-promise-allSettled
+/**
+ * This function wraps existing promises with the intention to allow the collection of promises
+ * to settle when used in conjunction with Promise.all(). Promise.all() by default executes until the first
+ * rejection. We are looking for the equivalent of Promise.allSettled() which is scheduled for ES2020.
+ * Once the functionality is  available  in the near future this function can be removed.
```

Comment:
When the functionality is available this function can be removed.

(Do you really want to promise "near future"?) :-)